test/langtools/lib/annotations/annotations/classfile/ClassfileInspector.java

```diff
@@ -1197,8 +1197,8 @@ public void annoMatcher(Attribute<?> attr, ExpectedAnnotation expected) {
         switch (attr) {
             case RuntimeVisibleTypeAnnotationsAttribute rvattr -> {
                 if (expected.matchVisibility(true)) {
-                    for(Annotation anno : rvattr.annotations()) {
-                        expected.matchAnnotation(anno);
+                    for(var anno : rvattr.annotations()) {
```

Comment:
```suggestion
                    for (var anno : rvattr.annotations()) {
```

Reply:
There are 43 `for(` and 2 `for (` in this file, so fixing a single occurrence isn't too helpful; for consistency I think you can create a dedicated RFE for running IDE formatters over all jdk/classfile tests. I will review it if you submit such a patch.